docs/concepts/workers.md

```diff
@@ -4,4 +4,52 @@ title: Workers
 sidebar_label: Workers
 ---
 
-A Worker is a service that hosts the [Workflow](/docs/concepts/workflows) and [Activity](/docs/concepts/activities) implementations.
+A Worker is a service that does the following:
+
+- Hosts executable [Workflow](/docs/concepts/workflows) and [Activity](/docs/concepts/activities) code.
```

Comment:
Should we add here that you can have separate workers for workflow and activities. And that a single worker can host both too?

Reply:
how about this

```suggestion
- Hosts executable [Workflow](/docs/concepts/workflows) and/or [Activity](/docs/concepts/activities) code. (Either can be hosted independently)
```